pkg/data/crd.go

```diff
@@ -39,6 +39,10 @@ func createCRDs(ctx context.Context, restConfig *rest.Config) error {
 			crd.FromGV(harvesterv1.SchemeGroupVersion, "VirtualMachineRestore", harvesterv1.VirtualMachineRestore{}),
 			crd.FromGV(harvesterv1.SchemeGroupVersion, "Preference", harvesterv1.Preference{}),
 			crd.FromGV(harvesterv1.SchemeGroupVersion, "SupportBundle", harvesterv1.SupportBundle{}),
+			// The BackingImage struct is not compatible with wrangler schemas generation, pass nil as the workaround.
+			// The expected CRD will be applied by Longhorn chart.
```

Comment:
can u please file an issue on the LH side to check if this is something they will need to make it compatible or the wrangler is better to support it?

Reply:
I think this is something to be improved in wrangler